package.json

```diff
@@ -53,5 +56,11 @@
   },
   "lint-staged": {
     "*.{js,css,md,ts,jsx,tsx,yml}": "prettier --write"
+  },
+  "bugs": {
+    "url": "https://github.com/TimeTac/js-client-library/issues"
+  },
+  "directories": {
+    "test": "tests"
```

Comment:
We don't have a "tests" directory anymore. I removed it because all tests are in the src folder.